index.html

```diff
@@ -234,6 +234,12 @@ <h2 id="project_tagline">eiffel-remrem-generate</h2>
 </pre></td></tr></table>
 </p>
 
+<h3>
+<a id="configuration" class="anchor" href="#configuration" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Configuration</h3>
```

Comment:
I think we need an example of starting the microservice with authentication. You can of course use dummy values.